app/models/mixins/supports_feature_mixin.rb

```diff
@@ -58,38 +58,40 @@ module SupportsFeatureMixin
   extend ActiveSupport::Concern
 
   QUERYABLE_FEATURES = {
-    :add_host                     => 'Add Host',
-    :associate_floating_ip    => 'Associate a Floating IP',
+    :add_host                   => 'Add Host',
+    :associate_floating_ip      => 'Associate a Floating IP',
     # FIXME: this is just a internal helper and should be refactored
-    :control                      => 'Basic control operations',
-    :cloud_tenant_mapping         => 'CloudTenant mapping',
-    :backup_create                => 'CloudVolume backup creation',
-    :backup_restore               => 'CloudVolume backup restore',
-    :cinder_service               => 'Cinder storage service',
-    :create_host_aggregate        => 'Host Aggregate Creation',
-    :swift_service                => 'Swift storage service',
-    :delete                       => 'Deletion',
-    :delete_aggregate             => 'Host Aggregate Deletion',
-    :disassociate_floating_ip     => 'Disassociate a Floating IP',
-    :discovery                    => 'Discovery of Managers for a Provider',
-    :evacuate                     => 'Evacuation',
-    :events                       => 'Query for events',
-    :launch_cockpit               => 'Launch Cockpit UI',
-    :live_migrate                 => 'Live Migration',
-    :migrate                      => 'Migration',
-    :provisioning                 => 'Provisioning',
-    :reboot_guest                 => 'Reboot Guest Operation',
-    :reconfigure                  => 'Reconfiguration',
-    :refresh_new_target           => 'Refresh non-existing record',
-    :regions                      => 'Regions of a Provider',
-    :remove_host                  => 'Remove Host',
-    :resize                       => 'Resizing',
-    :retire                       => 'Retirement',
-    :smartstate_analysis          => 'Smartstate Analaysis',
-    :snapshots                    => 'Snapshots',
-    :terminate                    => 'Terminate a VM',
-    :update_aggregate             => 'Host Aggregate Update',
-    :refresh_network_interfaces   => 'Refresh Network Interfaces for a Host',
+    :control                    => 'Basic control operations',
+    :cloud_tenant_mapping       => 'CloudTenant mapping',
+    :create                     => 'Creation',
+    :backup_create              => 'CloudVolume backup creation',
+    :backup_restore             => 'CloudVolume backup restore',
+    :cinder_service             => 'Cinder storage service',
+    :create_host_aggregate      => 'Host Aggregate Creation',
+    :swift_service              => 'Swift storage service',
+    :delete                     => 'Deletion',
```

Comment:
@blomquisg this one must be here from a different PR.  Another non-specific Feature.